services/user/CommonApi/common/packages/plugin-tester/ui/src/components/ServiceInput.tsx

```diff
@@ -13,19 +13,27 @@ export function ServiceInput({
   onPluginChange,
   onLoad,
 }: ServiceInputProps) {
+  const handleKeyDown = (e: React.KeyboardEvent) => {
+    if (e.key === "Enter") {
+      onLoad();
+    }
+  };
+
   return (
-    <div style={{ display: "flex", gap: "1rem", marginBottom: "1rem" }}>
+    <div className="service-input-container">
       <input
-        style={{ padding: "0.5rem" }}
+        className="service-input"
         placeholder="Service name"
         value={service}
         onChange={(e) => onServiceChange(e.target.value)}
+        onKeyDown={handleKeyDown}
       />
       <input
-        style={{ padding: "0.5rem" }}
+        className="service-input"
         placeholder="Plugin name"
         value={plugin}
         onChange={(e) => onPluginChange(e.target.value)}
+        onKeyDown={handleKeyDown}
       />
```

Comment:
Normally we'd wrap the two inputs in a `form` component with an `onSubmit` prop:

```ts
const onSubmit = (e: React.KeyboardEvent) => {
    e.preventDefault();
    onLoad();
}

<form className="service-input-container" onSubmit={onSubmit}>
    // inputs without the `onKeyDown` prop
    <button type="submit" className="common-button">Load</button>
</form>
```

When `Enter` is pressed, the form submit handler fires.

Reply:
It works as is, so whether you want to make that change or not is up to you.